### DISCUSSION TITLE
addition to tv wallpaper

### DISCUSSION THREAD

**Discussion by orianaaaa** [rocksdanister/lively#1725] [Ideas]
i feel a good addition to the tv wallpaper would be the addition of a lyric option either to the screen itself or as its own 3d entity around the tv, ive also found a bug where if two songs from the same album play the name of the song will not change

**Comment by rocksdanister** [comment]
Are there any free service that provides lyrics?

**Reply by orianaaaa** [reply]
im not sure, i was imagining with spotify it would be simular to how the wallpaper grabs the song name and album cover but instead from the lyric tab in spotify, it could be like a premium feature :D
